Mongo extension is not loaded

I have the following dockerfile

```dockerfile
FROM bref/php-82-fpm-dev

COPY --from=bref/extra-mongodb-php-82 /opt /opt

```
After I compile my dockerfile and run it from the command line, I get the following warning:

```bash
PHP Warning:  PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /opt/bref/extensions/mongodb.so (libssl.so.1.1: cannot open shared object file: No such file or directory), /opt/bref/extensions/mongodb.so.so (/opt/bref/extensions/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
PHP 8.2.4 (cli) (built: Apr  3 2023 09:43:33) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.4, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.4, Copyright (c), by Zend Technologies
```

When I run the command `ls -la` in the path `/opt/bref/extensions` the mongodb.so file is there.

```bash
bash-4.2# ls -la  /opt/bref/extensions/
total 15788
drwxr-xr-x 1 root root    4096 Apr  1 14:17 .
drwxr-xr-x 1 root root    4096 Apr  1 14:17 ..
-rw-r--r-- 1 root root  632968 Apr  3 09:44 apcu.so
-rw-r--r-- 1 root root  465600 Apr  3 09:44 blackfire.so
-rwxr-xr-x 1 root root 1969928 Apr  3 09:44 intl.so
-rw-r--r-- 1 root root 9435544 Apr  1 14:17 mongodb.so
-rwxr-xr-x 1 root root 1160880 Apr  3 09:44 opcache.so
-rwxr-xr-x 1 root root   37600 Apr  3 09:44 pdo_mysql.so
-rwxr-xr-x 1 root root   58368 Apr  3 09:44 pdo_pgsql.so
-rw-r--r-- 1 root root 2376304 Apr  3 09:44 xdebug.so

```
But for some reason it doesn't find that extension in that path

When I run the `php -m` command it also doesn't show the mongo extension being loaded.

```
bash-4.2# php -m
PHP Warning:  PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /opt/bref/extensions/mongodb.so (libssl.so.1.1: cannot open shared object file: No such file or directory), /opt/bref/extensions/mongodb.so.so (/opt/bref/extensions/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
[PHP Modules]
bcmath
Core
ctype
curl
date
dom
exif
fileinfo
filter
ftp
gettext
hash
iconv
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
posix
random
readline
Reflection
session
SimpleXML
soap
sockets
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache

```

same error when testing a custom layer

```dockerfile
FROM public.ecr.aws/lambda/provided:al2

COPY --from=bref/extra-mongodb-php-82:1.1.0 /opt /opt

... copy other extensions ...
```

output of testing:

```shell
###############################################
###############################################
### Testing layers/extensions PHP82
###
Sending build context to Docker daemon  2.048kB
Step 1/5 : ARG PHP_VERSION
Step 2/5 : ARG TARGET_IMAGE
Step 3/5 : FROM bref/$TARGET_IMAGE AS ext
 ---> aedbf57f0c8e
Step 4/5 : FROM bref/php-$PHP_VERSION:2
 ---> 0e38ce64966d
Step 5/5 : COPY --from=ext /opt /opt
 ---> f191daba7edc
Successfully built f191daba7edc
Successfully tagged bref/test-layers/extensions-82:latest

Warning: PHP Startup: Unable to load dynamic library 'mongodb.so' (tried: /opt/bref/extensions/mongodb.so (libssl.so.1.1: cannot open shared object file: No such file or directory), /opt/bref/extensions/mongodb.so.so (/opt/bref/extensions/mongodb.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
FAIL: Class "MongoDB\Driver\Manager" does not exist.
make: *** [test] Error 1
```

--- a/layers/mongodb/Dockerfile
+++ b/layers/mongodb/Dockerfile
@@ -9,6 +9,6 @@ RUN echo 'extension=mongodb.so' > /tmp/ext.ini
 # Build the final image with just the files we need
 FROM scratch
 
-# Copy things we installed to the final image
+# Copy the things we installed to the final image
 COPY --from=ext /tmp/mongodb.so /opt/bref/extensions/mongodb.so
 COPY --from=ext /tmp/ext.ini /opt/bref/etc/php/conf.d/ext-mongodb.ini

